pkg/config/remote/service/service.go

```diff
@@ -471,6 +476,14 @@ func NewService(cfg model.Reader, rcType, baseRawURL, hostname string, tagsGette
 
 	clock := clock.New()
 
+	// WebSocket test actor - must call Start() to spawn the background task.
+	var websocketTest startstop.StartStoppable
+	if cfg.GetBool("remote_configuration.no_websocket_echo") {
+		websocketTest = &noOpRunnable{}
+	} else {
+		websocketTest = NewWebSocketTestActor(http)
+	}
```

Comment:
My intention is to make it possible to disable this test, but I'm not familiar with how the config system works. 

Is this call sufficient to enable a user to disable the test or is there more config plumbing I need to do?

Reply:
Yeah, the only other alternative is adding a DD signed configurations to sync it to every org